### PR TITLE
A few minor things to clean up

### PR DESCRIPTION
**Pull request by garyb**
- The change to `runUI` to only accept `Component` is okay because `ComponentP` is only used by components-with-children that have not yet had their children installed.
- Renaming `ρ` to `r` for the phantom rows is unfortunate, but seems like the best option currently to ensure "everyone" can build Halogen
- Forgot to add the `interpret` example to the stuff travis runs when I did that PR

### CODE REVIEW

File: src/Halogen/Component.purs [hunk 1]
```diff
@@ -232,7 +232,15 @@ install :: forall s s' f f' g p p'. (Plus g, Ord p)
         => ParentComponent s s' f f' g p p'
         -> (p -> ChildState s' f' g p')
         -> InstalledComponent s s' f f' g p p'
-install c f = Component { render: render c f, eval: eval, peek: const (pure unit) }
+install c f = installWithState c (const f)
+
+-- | A version of [`install`](#install) that gives us access to the parent's
+-- | state while installing children.
+installWithState :: forall s s' f f' g p p'. (Plus g, Ord p)
```
Comment (cryogenian) [inline src/Halogen/Component.purs]: 'installWitState' is not exported

Comment (garyb) [inline src/Halogen/Component.purs]: Oops! Thanks, I'll put a fix in later today.

**Comment by garyb** [comment]
Does anyone have any other comments or can I merge this now? I've added a few more changes since the original PR, outlined in the commit comments.

/cc @beckyconning @cryogenian @jonsterling

**Comment by jonsterling** [comment]
I think it looks good

**Comment by garyb** [comment]
Cool, thanks :smile:

**Pull request merged**